docs/upgrading.md

```diff
@@ -0,0 +1,30 @@
+# Upgade Guide
+
+## Configuration Changes (Q1 2023)
+
+`polis.config.template.js` and `polis.config.js` files are removed and no longer used.
+`docker-dev.env` and `docker-db-dev.env` files are removed and no longer used.
+`.env` and/or `prod.env` are now treated as the source of truth for the application and are ignored by git.
+See `example.env` for default values.
+
+Please read [configuration.md](./configuration.md) for more information and a complete list of configuration values.
+
+Values that have been renamed or replaced:
+
+- **`DATABASE_FOR_READS_NAME`** has been replaced by **`READ_ONLY_DATABASE_URL`**
```

Comment:
@ballPointPenguin This is somewhat problematic with usage on Heroku, since the main `DATABASE_URL` (and any read replica dbs) needs to be managed by the system. That's why we implemented `DATABASE_FOR_READS_NAME` as a pointer to the var that gets read from for the read only database.

Reply:
Lets fork this off as a separate issue

Reply:
https://github.com/compdemocracy/polis/issues/1671